riptide-failsafe/src/test/java/org/zalando/riptide/failsafe/RetryAfterDelayFunctionTest.java

```diff
@@ -62,11 +60,7 @@
                     .withRetryPolicy(new RetryPolicy()
                             .withDelay(2, SECONDS)
                             .withDelay(new RetryAfterDelayFunction(clock))
-                            .withMaxRetries(4))
-                    .withCircuitBreaker(new CircuitBreaker()
-                            .withFailureThreshold(3, 10)
-                            .withSuccessThreshold(5)
-                            .withDelay(1, TimeUnit.MINUTES)))
+                            .withMaxRetries(4)))
```

Comment:
This change did not break any test, didn't it?

Reply:
Wasn't used here. Actually I'm missing a proper CircuitBreakerTest

Reply:
Added test